daemon/stats/collector.go

```diff
@@ -106,6 +108,8 @@ func (s *Collector) Run() {
 				logrus.Errorf("collecting stats for %s: %v", pair.container.ID, err)
 			}
 		}
+
+		time.Sleep(s.interval)
```

Comment:
Why did you use `time.Sleep` and not kept using `range time.Tick`?
Unless I am wrong, they are similar in functionality.

Reply:
If it takes more than the interval time, the ticks backup and cause a burst of sampling, making the issue worse.